Makefile

```diff
@@ -14,6 +14,11 @@ VERSION := 4.13
 OPERATOR_NAMESPACE 			:= runoncedurationoverride-operator
 OPERATOR_DEPLOYMENT_NAME 	:= runoncedurationoverride-operator
 
+CODEGEN_OUTPUT_PACKAGE :=github.com/openshift/run-once-duration-override-operator/pkg/generated
```

Comment:
Who is going to use these variables?

Reply:
Needed by targets/openshift/codegen.mk, resp. targets/openshift/crd-schema-gen.mk